mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -227,6 +227,7 @@ def _get_write_credential_infos(self, remote_file_paths):
             GetCredentialsForWrite, self.run_id, run_relative_remote_paths
         )
 
+    @override
```

Comment:
why do we need this?

Reply:
As artifact_repo implements default download_trace_data and upload_trace_data methods, apply @override here for annotating we're overriding these functions here

Reply:
I'd use this if we used `@override` for other overridden methods.

Reply:
Importing override also requires typing_extensions version >= some version, I'll remove it instead then :)